src/nplinker/arranger.py

```diff
@@ -27,17 +27,23 @@
 
 
 class DatasetArranger:
-    """Arrange the dataset required by NPLinker.
+    """Arrange datasets based on the fixed working directory structure with the given configuration.
 
-    This class is used to arrange the datasets required by NPLinker according to the
-    configuration. The datasets include MIBiG, GNPS, antiSMASH, and BiG-SCAPE.
+    ??? info "Concept and Diagram"
```

Comment:
Is this a syntax that mkdocs recognizes and links to "Concept and Diagram"?

Reply:
No, that syntax is not for link but for creating a info block (or callout), [looking like](https://nplinker.github.io/nplinker/latest/api/arranger/)

![image](https://github.com/user-attachments/assets/db77b5ec-7983-4e68-867c-5637e6de0df5)